test/controllers/visual_groups_controller_test.rb

```diff
@@ -98,8 +98,8 @@ class VisualGroupsControllerTest < FunctionalTestCase
     patch(:update, params: {
             id: @visual_group.id,
             visual_group:
-              { name: "",
-                approved: @visual_group.approved }
+            { name: "",
+              approved: @visual_group.approved }
```

Comment:
This doesn't seem like the best way to do this and there is another case in the same file.  In my PR I just updated these cases to look like:
```
            visual_group: {
              name: @visual_group.name,
              approved: @visual_group.approved
            }
```
which Rubocop is also fine with and is more consistent with another spot in the file and seems cleaner to me.